Adaptors/Redis/src/ObjectStorage.cs

```diff
@@ -83,21 +85,22 @@ await taskList.WhenAll()
                                      IAsyncEnumerable<ReadOnlyMemory<byte>> valueChunks,
                                      CancellationToken                      cancellationToken = default)
   {
-    using var _ = logger_.LogFunction(objectStorageName_ + key);
+    var       storageNameKey = objectStorageName_ + key;
+    using var _              = logger_.LogFunction(storageNameKey);
 
     var idx      = 0;
     var taskList = new List<Task>();
     await foreach (var chunk in valueChunks.WithCancellation(cancellationToken)
                                            .ConfigureAwait(false))
     {
-      taskList.Add(redis_.StringSetAsync(objectStorageName_ + key + "_" + idx,
-                                         chunk));
+      var idx1 = idx;
+      taskList.Add(PerformActionWithRetry(() => redis_.StringSetAsync(storageNameKey + "_" + idx1,
```

Comment:
si tu utilises une $"" string tu n'auras peut etre pas besoin de faire la copie ici.
Du coup, ça serait pas mal de l'utiliser partout

Reply:
I removed uidx1 but I construct string outside the tasklist.add
This is to avoid this error : 
![image](https://user-images.githubusercontent.com/113617172/219371087-53ea2354-02e3-4f27-8d6c-f7d79a36a372.png) not sure if it can imply index mix and bug. so I followed resharper advice